pkg/agent/openflow/network_policy.go

```diff
@@ -738,8 +744,8 @@ func (c *client) InstallPolicyRuleFlows(ruleID uint32, rule *types.PolicyRule, n
 	if err := c.applyConjunctiveMatchFlows(ctxChanges); err != nil {
 		return err
 	}
-	// Add the policyRuleConjunction into policyCache.
-	c.policyCache.Store(ruleID, conj)
+	// Add the policyRuleConjunction into policyCache
```

Comment:
Actually it's correct to make the comment full sentences.
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Fixed